lib/schemas.rkt

```diff
@@ -0,0 +1,201 @@
+#lang racket
+
+(provide #;(rename-out [yomi-module #%module-begin])
```

Comment:
Disabled the custom `#%module-begin` while testing. IMO the macros now do a better job of safety checks than this does now. This could be removed with a slight tweak to how button identifiers get built in `define-game`; define the buttons with the `button:` prefix attached instead of exporting them with the prefix. This gives you the versatility of being able to define game/character schemas in the same file, although we consider this to be a bad practice.